networkx/algorithms/dominance.py

```diff
@@ -135,3 +135,37 @@ def dominance_frontiers(G, start):
                         df[v].add(u)
                         v = idom[v]
     return df
+
+
+def dominator_tree(G, start):
+    """Returns the dominator tree from a directed graph.
+
+
+    Parameters
+    ----------
+    G : a DiGraph or MultiDiGraph
+        The graph where dominance is to be computed.
+
+    start : node
+        The start node of dominance computation.
+
+    Returns
+    -------
+    T : DiGraph
+        Dominator tree where each node's children are those it immediately dominates
+
+    Raises
+    ------
+    NetworkXNotImplemented
+        If `G` is undirected.
+
+    NetworkXError
+        If `start` is not in `G`.
+
+    Notes
+    -----
+    Note that if a CFG has a single exit node,
```

Comment:
Is this note relevant to this particular function? It doesn't seem to mention a dominator tree anywhere. If it is relevant, can you rephrase it so that it applies to any directed graph (since this function is generic)?